ai_feedback/models/DeepSeekV3Model.py

```diff
@@ -44,18 +46,24 @@ def generate_response(
             test_output (Optional[Path]): Path Object pointing to the test output file.
             llama_mode (Optional[str]): Optional mode to invoke llama.cpp in.
             question_num (Optional[int]): An optional question number to target specific content.
+            json_schema (Optional[str]): Optional json schema to use.
 
         Returns:
             Optional[Tuple[str, str]]: A tuple containing the prompt and the model's response,
                                        or None if the response was invalid.
         """
+        schema = None

```

Comment:
Let's keep the code structure consistent with the other classes (if-else, etc.) Also keep the `open` call the same, with the `utf-8` encoding